apphosting.yaml

```
@@ -0,0 +1,12 @@
+runConfig:
+  minInstances: 0
+  maxInstances: 10
+  cpu: 2
+  memoryMiB: 512
+
+env:
+  - variable: MAPTILER_KEY
+    secret: projects/766398966649/secrets/maptiler-key
```

You might not need the full path?

```suggestion
    secret: maptiler-key
```

---

I was basing this on what Paul Xu did in ChatCCV

https://github.com/brown-ccv/chat-ccv/blob/196442747599472df402fbdf237ee1954df7435e/apphosting.yaml#L10-L14

If I don't need it, I'm stoked, but I dont know 😓